frameworks/AutoPyTorch/setup.sh

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env bash
+
+HERE=$(dirname "$0")
+VERSION=${1:-"stable"}
+REPO=${2:-"https://github.com/automl/Auto-PyTorch.git"}
+PKG=${3:-"autoPyTorch"}
+if [[ "$VERSION" == "latest" ]]; then
+    VERSION="master"
+fi
+
+OS_NAME=$(cat /etc/os-release | grep '^NAME' | cut -d '"' -f 2)
+OS_VERSION=$(cat /etc/os-release | grep '^VERSION_ID' | cut -d '"' -f 2)
+OS="${OS_NAME} ${OS_VERSION}"
+OS_SUPPORTED="Amazon Linux 2,Ubuntu 18.04,Ubuntu 20.04"
+
+if echo "${OS_SUPPORTED}" | tr "," "\n" | grep -F -x -q "${OS}"; then
+    echo "INFO: Operating system ${OS} is supported."
+else
+    echo "ERROR: Operating system ${OS} is not supported."
+    exit 1
+fi
+
+# creating local venv
+. ${HERE}/../shared/setup.sh ${HERE} true
+
+PIP install --upgrade pip
+PIP install --upgrade setuptools wheel
+
+
+
+
+if [[ "${OS}" == "Amazon Linux 2" ]]; then
+    # TOPIC: update system
+    echo "updating system..."
+    SUDO yum clean all
+    SUDO rm -rf /var/cache/yum
+    SUDO yum -y update
+
+#     # TOPIC: install readline-devel (for rpy2)
+#     echo "installing readline-devel..."
+#     SUDO  yum install -y readline readline-devel
+#     # ---------------------
+
+#     # TOPIC: install R
+#     echo "installing R..."
+#     SUDO  yum install -y gcc make sqlite-devel zlib-devel libffi-devel openssl-devel bzip2-devel wget tar gzip
+#     SUDO  yum install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
+#     SUDO  yum install -y R
+#     # R version=3.6.0
+#     SUDO  yum -y install libcurl-devel
+#     SUDO  R -e 'install.packages(c("forecast"), repos="https://cloud.r-project.org")'
+
+elif [[ "${OS}" == "Ubuntu 18.04" || "${OS}" == "Ubuntu 20.04" ]]; then
+    # TOPIC: update system
+    echo "updating system..."
+    DEBIAN_FRONTEND=noninteractive
+    SUDO apt-get update -y
+    SUDO apt-get upgrade -y
+    # SUDO apt-get install -y apt-utils make cmake build-essential libssl-dev zlib1g-dev libbz2-dev libreadline-dev \
+    #     libsqlite3-dev wget curl llvm libncurses5-dev xz-utils tk-dev libxml2-dev libxmlsec1-dev \
+    #     libffi-dev liblzma-dev libcurl4-openssl-dev p7zip-full awscli python3-dev
+    # ------------------
+
+#     # TOPIC: install R
+#     echo "installing R..."
+#     # forecast requires R 3.6 (Which is not available for Ubuntu 18.04 by default.)
+#     if [[ "${OS}" == "Ubuntu 18.04" ]]; then
+#         SUDO apt install -y software-properties-common
+#         SUDO apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys E298A3A825C0D65DFD57CBB651716619E084DAB9
+#         SUDO add-apt-repository 'deb https://cloud.r-project.org/bin/linux/ubuntu bionic-cran35/'
+#         SUDO apt-get -y update
+#     fi
+#     apt install -y r-base
+#     SUDO R -e 'install.packages(c("forecast", "nnfor"), repos="https://cloud.r-project.org")'
+#     # ---------------------
+    SUDO apt-get install -y python3-opencv
```

Comment:
Just to double-check, is `python3-opencv` needed (for time series)? Why isn't it installed on the other OS?